src/pages/components/Navbar/SubMenu.js

```diff
@@ -99,7 +104,7 @@ const SubMenu = ({ isVisible, transparent }) => {
               backgroundColor: 'transparent',
             }}
           >
-            {language === 'en' ? '🇺🇸' : '🇳🇬'}
+            {flags.get(language) ? flags.get(language) : '' }
```

Comment:
the case where ` flags.get(language) === undefined ` should **never**  happen. If it does, it's either i18next's fault or the fault of a lazy developer who added a translation. For now, it should be enough to leave all these small things to be changed manually, in future there should be a way of keeping it all in one place (e.g. the language's JSON-File), making it easier to add translations!